assets/icons/chevron-right.svg

```diff
@@ -1,3 +1,3 @@
 <svg viewBox="0 0 16 16" fill="none" xmlns="http://www.w3.org/2000/svg">
-<path d="M6 12L10 8L6 4" stroke="white" stroke-width="1.5" stroke-linecap="round" stroke-linejoin="round"/>
+<path d="M6 12L10 8L6 4" stroke="currentColor" stroke-width="1.5" stroke-linecap="round" stroke-linejoin="round"/>
```

Comment:
All the `chevron-right.svg` lose their white color.
We must set the `color` prop to many `SVG` so
![image](https://github.com/TERITORI/teritori-dapp/assets/50441633/1fa65a7f-4d6d-4dd1-9359-0b3c83e22c9e)

Reply:
Yep ty i did that on some other svg files 👍

Reply:
Oh but i think you didn't finished the review because i did that in newer commits 👍